R/post-tweet.R

```diff
@@ -131,15 +131,15 @@ post_tweet <- function(status = "my first rtweet #rstats",
   ## geotag if provided
   if (!is.null(lat) && !is.null(long)) {
     # Validate inputs
-    if (!is.numeric(lat)) stop("lat should be numeric")
-    if (!is.numeric(long)) stop("long should be numeric")
+    if (!is.numeric(lat)) stop("`lat` should be numeric.")
```

Comment:
Sorry, just realised that these would be better as "must", not "should"

Reply:
No worries, done